hafnian/_hafnian.py

```diff
@@ -16,7 +16,7 @@
 """
 import numpy as np
 
-from .lib.libhaf import haf_complex, haf_real, haf_int, haf_rpt_real, haf_rpt_complex
+from .lib.libhaf import haf_complex, haf_real, haf_int, haf_rpt_real, haf_rpt_complex, perm_complex, perm_real
```

Comment:
How come `perm_complex` and `perm_real` are being imported here? If they are not being used in this module, they can be deleted